app/views/tag/_tags.html.erb

```diff
@@ -1,47 +1,53 @@
 <% user = user || @user # allow overriding w/ local variable %>
-<ul class= "list-inline">
-  <% tags.each do |tag| %>
-    <% if tag.class == NodeTag %>
-      <% if power_tag ^ tag.name.include?(':') # XOR operator??? %>
-        <li style="width: 100%;"><span id="tag_<%= tag.tid %>" class="badge <%= badge_name %> pop" style="cursor:pointer" data-toggle="popover" data-trigger="focus" data-count=0 data-placement="top" data-content="<p style='text-align:center;'><a href='/tag/<%= tag.name %>'><%= Tag.tagged_node_count(tag.name) || 0 %> notes</a> - <a href='/contributors/<%= tag.name %>'><%= Tag.contributors(tag.name).count %> people <br></a></p> <p style='text-align:center;font-size:12px;'><%if tag.description %><%= tag.description %> |<% end %> created by <a href='/profile/<%= tag.try(:author).try(:username) %>'><%= tag.try(:author).try(:username) %></a> <%= time_ago_in_words(Time.at(tag.date)) %> ago </p><div style='text-align:center;' class='text-center'><a href='/subscribe/tag/<%= tag.name %>' class='btn btn-primary'>Follow</a></div>" data-html="true" title="<%= tag.name %>">
-         <%= tag.name %>
-          <% if logged_in_as(['admin', 'moderator']) || (current_user && ( current_user.uid ==  @node.uid || current_user.uid == tag.uid)) %>
-            <% if tag.name.include? ':' %>
-              <a data-confirm="This is a power tag (see https://publiclab.org/wiki/power-tags) -- and may drive a specific function on this page. Are you sure you want to delete it?" class="tag-delete" data-remote="true" href="/tag/delete/<%= @node.id %>/<%= tag.tid %>" data-tag-id="<%= tag.tid %>" data-method="delete" style="color:white;">x</a>
-            <% else %>
-              <a data-confirm="Are you sure you want to delete it?" class="tag-delete" data-remote="true" href="/tag/delete/<%= @node.id %>/<%= tag.tid %>" data-tag-id="<%= tag.tid %>" data-method="delete" style="color:white">x</a>
-            <% end %>
-          <% end %>
-        </span></li>
-      <% end %>
-
-    <% elsif tag.class == UserTag && (tag.name[0..4] != "oauth" || (logged_in_as(['admin', 'moderator']) || (current_user && current_user.uid == tag.uid))) %>
+<% tags.limit(2).each do |tag| %>
```

Comment:
Here, I think this may cause some trouble - it looks like we would need to switch by `tag.class` outside this iterator, so profile tags are unaffected. Then, we can split off the first 2, within the `NodeTag` block, and show those as cards. The rest we can show, still in that block, the previous way.